drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -91,7 +93,13 @@ local function process_rest_response(response, err, partial)
 end
 
 function PhilipsHueApi.new_bridge_manager(base_url, api_key, socket_builder)
+  log.debug_with({hub_logs = true}, st_utils.stringify_table(
```

Comment:
Do we want this at info level so it will be visible on test hubs?

Reply:
No, I won't want to leak API keys even though you can't do much with them. This was truly a debug print. I'd like to leave that in for the driver-side logs though.